tests/D2L.CodeStyle.Analyzers.Test/IGlobalContext/IGlobalContextAnalyzerTests.cs

```diff
@@ -0,0 +1,117 @@
+﻿using D2L.CodeStyle.Analyzers.Test.Verifiers;

```

Comment:
Here are the cases of `IGlobalContext` usage I find, I may miss something, please let me know